examples/create_checks_from_csv/check.py

```diff
@@ -0,0 +1,183 @@
+# usage:
+#   python check.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide your API Key, keep this secure
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'
+
+# TODO: Create your from_address
+try:
+    from_address = lob.Address.create(
+        name='Your Name',
+        company='Your Company',
+        address_line1='123 Test Avenue',
+        address_line2='STE 456',
+        address_city='San Francisco',
+        address_state='CA',
+        address_zip='94107'
+    )
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create from_address.')
+    sys.exit(1)
+
+# TODO: Create and verify your bank account
+try:
+    bank_account = lob.BankAccount.create(
+        description='Your Bank Account',
+        routing_number='122100024',
+        account_number='0123456478',
+        account_type='company',
+        signatory='John Hancock'
+    )
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create bank account')
+    sys.exit(1)
+
+try:
+    example_bank_account = lob.BankAccount.verify(
```

Comment:
nit: this is short enough that it can be in one line
```python
example_bank_account = lob.BankAccount.verify(id=bank_account.id, amounts=[23, 77])
```